4-projects/README.md

```diff
@@ -9,42 +9,79 @@ The purpose of this step is to setup folder structure and projects for applicati
-1. Rename nonprod.auto.example.tfvars to nonprod.auto.tfvars and update the file with the perimeter_name that starts with `sp_n_shared_restricted`.
-1. Rename prod.auto.example.tfvars to prod.auto.tfvars and update the file with the perimeter_name that starts with `sp_p_shared_restricted`.
+1. Rename `common.auto.example.tfvars` to `common.auto.tfvars` and update the file with values from your environment and bootstrap.
+1. Rename `dev.auto.example.tfvars` to `dev.auto.tfvars` and update the file with the `perimeter_name` that starts with `sp_d_shared_restricted`.
+1. Rename `nonprod.auto.example.tfvars` to `nonprod.auto.tfvars` and update the file with the `perimeter_name` that starts with `sp_n_shared_restricted`.
+1. Rename `prod.auto.example.tfvars` to `prod.auto.tfvars` and update the file with the `perimeter_name` that starts with `sp_p_shared_restricted`.
 1. Commit changes with `git add .` and `git commit -m 'Your message'`
 1. Push your plan branch to trigger a plan `git push --set-upstream origin plan` (the branch `plan` is not a special one. Any branch which name is different from `dev`, `nonprod` or `prod` will trigger a terraform plan).
     1. Review the plan output in your cloud build project https://console.cloud.google.com/cloud-build/builds?project=YOUR_CLOUD_BUILD_PROJECT_ID
-1. Merge changes to dev with `git checkout -b dev` and `git push origin dev`
+1. Merge changes to dev branch with `git checkout -b dev` and `git push origin dev`
     1. Review the apply output in your cloud build project https://console.cloud.google.com/cloud-build/builds?project=YOUR_CLOUD_BUILD_PROJECT_ID
-1. Merge changes to nonprod with `git checkout -b nonprod` and `git push origin nonprod`
+1. Merge changes to nonprod branch with `git checkout -b nonprod` and `git push origin nonprod`
     1. Review the apply output in your cloud build project https://console.cloud.google.com/cloud-build/builds?project=YOUR_CLOUD_BUILD_PROJECT_ID
-1. Merge changes to prod with `git checkout -b prod` and `git push origin prod`
+1. Merge changes to prod branch with `git checkout -b prod` and `git push origin prod`
     1. Review the apply output in your cloud build project https://console.cloud.google.com/cloud-build/builds?project=YOUR_CLOUD_BUILD_PROJECT_ID
 
 
+### Setup to run via Jenkins
+1. Clone the repo you created manually in bootstrap: `git clone <YOUR_NEW_REPO-4-projects>`
+1. Navigate into the repo `cd YOUR_NEW_REPO_CLONE-4-projects` and change to a non prod branch `git checkout -b plan` (the branch `plan` is not a special one. Any branch which name is different from `dev`, `nonprod` or `prod` will trigger a terraform plan).
+1. Copy contents of foundation to new repo `cp -R ../terraform-example-foundation/4-projects/* .` (modify accordingly based on your current directory).
+1. Copy the `Jenkinsfile` to the root of your new repository (modify accordingly based on your current directory):
```

Comment:
I think these instruction cloud use the same format/style used in the cloud build instructions for consistency